apps/prairielearn/src/migrations/20250815145453_course_instance__join_id__add.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE course_instances
+ADD COLUMN join_id VARCHAR(255);
```

Comment:
Typically we want to reserve names like `*_id` for foreign keys. In PT we use `self_enrollment_key` for this. Even in the PR title you call this "self-enrollment join code". Can we change this?

I don't have a strong preference for `self_enrollment_key` vs. `join_key` vs. `join_code` vs. something similar, though I have a slight preference for something that explicitly mentions "enrollments" here.

Reply:
I'm thinking `enrollment_code` might be a good one.

Reply:
Done in [3120d8a](https://github.com/PrairieLearn/PrairieLearn/pull/12647/commits/3120d8af037426e69c16642012fc799e8091466c).